material-overrides/assets/stylesheets/polkadot.css

```diff
@@ -1513,34 +1513,74 @@ div.mermaid {
   font-size: 12px;
 }
 
-/** Where to Go Next Badge styling */
-.md-typeset span.badge {
+/** Badge Styling */
+/* Basic style */
+span.badge {
   border-radius: 0.5em;
   font-size: 0.8em;
   padding: 0.2em 0.5em;
-  margin-right: 0.5em;
 }
 
-.md-typeset span.badge.tutorial {
+/* Color Variants */
+span.badge.tutorial {
   background-color: var(--electric-violet);
   color: var(--white);
 }
-
-.md-typeset span.badge.guide {
+span.badge.guide {
   background-color: var(--chartreuse);
   color: var(--black);
 }
-
-.md-typeset span.badge.learn {
+span.badge.learn {
   background-color: var(--aqua);
   color: var(--black);
 }
-
-.md-typeset span.badge.external {
+span.badge.external {
   background-color: var(--blue-ribbon);
   color: var(--white);
 }
 
+span.badge.beginner {
+  background-color: rgb(32, 172, 32);
+  color: var(--white);
+}
+span.badge.intermediate {
+  background-color: var(--chartreuse);
+  color: var(--black);
+}
+span.badge.advanced {
+  background-color: red;
+  color: var(--white);
+}
+
+/* Cards "In this Section" */
+.subsection-wrapper .card span.badge {
+  display: inline-block;
+  margin-top: 0.9em;
+  margin-left: 1.5em;
+}
+
+/* Cards "Where to Go Next" */
+.grid.cards span.badge {
+  margin-right: 0.5em;
+}
```

Comment:
All of the styling for the content page cards should be moved next to the other card styles we previously have defined so everything card-related is in one area of the css. Everything for index page cards should be in the index page css file